packages/searchbox/tests/site/index.html

```diff
@@ -0,0 +1,378 @@
+						<div class="au-header__text">
+							<h1 class="au-header__heading">Site title</h1>
+							<div class="au-header__subline">
+								Service description that could be a little longer
+							</div>
+						</div>
+					</a>
+				</div>
+				<div class="col-md-3">
+					<form role="search"  class="au-search">
+						<label for="standard-nav" class="au-search__label">Search</label>
+						<input type="search" id="standard-nav" name="standard" class="au-text-input" />
+						<div class="au-search__btn">
+							<button class="au-btn"><spaform role="search"lass="au-search__submit-btn-text">Search</span></button>
+						</div>
+					</form>
+
+				</div>
+			</div>
+		</div>
+	</header>
+	<nav class="au-main-nav" aria-label="main">
+		<div class="container">
+			<div class="row">
+				<div class="col-md-12">
+					<div id="main-nav-default-3" class="au-main-nav__content">
+						<button
+							aria-controls="main-nav-default"
+							aria-expanded="false"
+							class="au-main-nav__toggle au-main-nav__toggle--open"
+							onClick="return AU.mainNav.Toggle( this )">
+							Menu
+						</button>
+						<div class="au-main-nav__menu">
+							<div class="au-main-nav__menu-inner">
+								<div class="au-main-nav__focus-trap-top"></div>
+								<button
+									aria-controls="main-nav-default"
+									class="au-main-nav__toggle au-main-nav__toggle--close"
+									onClick="return AU.mainNav.Toggle( this )">
+									Close
+								</button>
+								<ul class="au-link-list">
+									<li><a href="#">About</a></li>
+									<li><a href="#">Get started</a></li>
+									<li class="active"><a href="#" aria-current="page">Components</a></li>
+									<li><a href="#">Templates</a></li>
+									<li><a href="#">Community</a></li>
+									<li><a href="#">Support</a></li>
+								</ul>
+								<div class="au-main-nav__focus-trap-bottom"></div>
+							</div>
+						</div>
+						<div
+							class="au-main-nav__overlay"
+							aria-controls="main-nav-default"
+							onClick="return AU.mainNav.Toggle( this )">
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</nav>
+	<!-- END HEADER WITH SEARCH BOX PLAIN-->
+	<br><br><br>
+
+	<!-- HEADER WITH SEARCH BOX DARK-->
+
+	<h1> AU header with search DARK</h1>
+	<header class="au-grid au-header au-header--dark" role="banner">
+		<div class="container-fluid">
+			<div class="row">
+				<div class="col-md-8">
+					<a class="au-header__brand" href="#">
+						<img class="au-header__brand-image" alt="Insert alternate text here" src="https://designsystem.gov.au/assets/img/placeholder/256x80.png">
+						<div class="au-header__text">
+							<h1 class="au-header__heading">Site title</h1>
+							<div class="au-header__subline">
+								Service description that could be a little longer
+							</div>
+						</div>
+					</a>
+				</div>
+				<div class="col-md-4">
+					<form role="search" class="au-search au-search--dark">
```

Comment:
```suggestion
					<form role="search"  aria-label="Sitewide" class="au-search">
```

Refer: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Search_role#Labeling_landmarks

Reply:
sure, won't commit this since a class was removed